commerce/domain/tax/service/classes/TaxServiceExtensionResolverSample.cls

```diff
@@ -0,0 +1,36 @@
+// This sample is for the situation when different tax behaviors need to be implemented for different locales.
+// These tax behaviors can be -
+// 1. ResolutionState - EXECUTE_DEFAULT (the default Salesforce Internal Tax Api),
+// 2. ResolutionState - EXECUTE_REGISTERED (extended or overridden implementations via the extension point from the default Salesforce Internal Tax Api)
+// 3. ResolutionState - OFF (In this case, the default Salesforce Internal Tax Api will return an empty response).
+// For Salesforce Internal Tax calculation, please see the corresponding documentation.
+
+// An Extension Provider is a custom apex class which extends or overrides the default Salesforce Internal Tax Api.
+// An Extension Resolver is a custom apex class which selects different resolution states (EXECUTE_DEFAULT, EXECUTE_REGISTERED and OFF) for different locales 
+// to execute respective implementations (Extension Providers or the Default Salesforce Internal Tax Api).
+
+// Your custom apex extension providers and the resolver must be registered with the tax extension point and then the resolver must be registered and mapped to the web store via appropriate setup.
```

Comment:
It is unclear what resolver registration is required and mapped to the web store means.  Please provide a link to public documentation.